src/widgets/lv_dropdown.c

```diff
@@ -6,7 +6,12 @@
 /*********************
  *      INCLUDES
  *********************/
+#ifdef ARDUINO
+   #include "../core/lv_obj.h"
```

Comment:
Only this line is required. `<lvgl/src/core/lv_obj.h>` was added by accident.

Reply:
#if defined(LV_LVGL_H_INCLUDE_SIMPLE) || defined( ARDUINO ):
as long as LV_LVGL_H_INCLUDE_SIMPLE is defined in the Arduino ecosystem (e.g. in platform.local.txt) it works without the ARDUINO flag.

lv_conf_internal.h: with the update it works

Because it belongs to the topic: in lv_examples I have this construct several times because of the strange handling of #include directives by Arduino:
_#ifdef ARDUINO
   #include <lv_demo.h>
#else
   #include <lv_examples/lv_demo.h>
#endif_
Do you have an idea how to solve it more elegantly?

Reply:
> lv_conf_internal.h: with the update it works

Great! I added it here: a995695c


>Because it belongs to the topic: in lv_examples I have this construct several times because of the strange handling of #include directives by Arduino:

Do you think we can do the same trick as with` LV_LVGL_H_INCLUDE_SIMPLE`? E.g. `LV_DEMO_H_INCLUDE_SIMPLE`?

We can add this to `lv_demo.h`:
```c#if defined __has_include
#  if __has_include("lv_demo.h")
#   ifndef LV_DEMO_H_INCLUDE_SIMPLE
#    define LV_DEMO_H_INCLUDE_SIMPLE
#   endif
#  endif
#endif
```

Reply:
Sorry I didn't think of it before, it currently only affects [https://github.com/lvgl/lv_examples](https://github.com/lvgl/lv_examples). There I've just replaced it with #include "../lv_demo.h". Is probably the simplest solution.
With this, I am now ARDUINO "free".